browser/ui/brave_browser.cc

```diff
@@ -49,6 +53,29 @@ void BraveBrowser::ScheduleUIUpdate(content::WebContents* source,
 #endif
 }
 
+bool BraveBrowser::ShouldDisplayFavicon(
+    content::WebContents* web_contents) const {
+  // Override to not show favicon for NTP in tab.
+
+  // Suppress the icon for the new-tab page, even if a navigation to it is
```

Comment:
Re-added NTP checking condition that deleted by https://chromium-review.googlesource.com/c/chromium/src/+/2785855